Sources/TensorFlow/Layer.swift

```diff
@@ -191,3 +191,35 @@ public final class Parameter<Scalar: TensorFlowScalar> {
         self.value = value
     }
 }
+
+/// A wrapper around a differentiable value with toggleable trainability.
+@_propertyWrapper
+public struct Trainable<Value: Differentiable> : Differentiable {
```

Comment:
As discussed in person, I feel that `Trainable` could give the wrong intuition that only properties tagged `@Trainable` can be trained. It's better to call it `@Freezable` or something.

Reply:
I agree `@Freezable` is better! The flag is now called `isFrozen` and defaults to false instead of true.